controlplane/hack/deploy/addon/managed-serviceaccount/README.md

```diff
@@ -0,0 +1,15 @@
+# Install managed-serviceaccount add-on 
+
+1. Install manifests on the standalone cluster
```

Comment:
could we deploy together? like deploy controlplane+addon in one script

Reply:
`make deploy-all` can do this